appengine-managed-runtime/pom.xml

```diff
@@ -31,6 +31,7 @@
   <properties>
     <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
     <appengine.api.version>1.9.40</appengine.api.version>
+    <gcloud.api.version>0.8.2-beta-SNAPSHOT</gcloud.api.version>
```

Comment:
Why are you using the -SNAPSHOT ??

You do need to initialize Cloud Logging and turn on the JUL hook to use it. -- I don't see that here.